main/webapp/modules/core/scripts/reconciliation/recon-dialog.js

```diff
@@ -45,43 +46,29 @@ ReconDialog.prototype._createDialog = function() {
 
   this._elmts = DOM.bind(dialog);
   this._elmts.dialogHeader.text($.i18n('core-recon/recon-col')+' "' + this._column.name + '"');
-  
+
   this._elmts.servicePanelMessage.html($.i18n('core-recon/pick-service'));
-  this._elmts.serviceListTitle.html($.i18n('core-recon/service-title'));
   this._elmts.addStandardServiceButton.html($.i18n('core-buttons/add-std-svc')+"...");
-  this._elmts.reconcileButton.html($.i18n('core-buttons/start-recon'));
   this._elmts.cancelButton.html($.i18n('core-buttons/cancel'));
   this._elmts.discoverServicesButton.html($.i18n('core-buttons/discover-services'));
+  this._elmts.nextButton.html($.i18n('core-buttons/next-dialog'));
 
   this._elmts.addStandardServiceButton.on('click',function() { self._onAddStandardService(); });
 
-  this._elmts.reconcileButton.on('click',function() { self._onOK(); });
   this._elmts.cancelButton.on('click',function() { self._dismiss(); });
+  this._elmts.nextButton.on('click',function() { self._nextDialog(self._column,self._selectedServiceRecordIndex,self._serviceRecords,self._record); });
 
   this._level = DialogSystem.showDialog(dialog);
   this._populateDialog();
-  this._registerDialogServiceOpener();
+
 };
 
-ReconDialog.prototype._registerDialogServiceOpener = function() {
-  var self = this;
 
-  $('.recon-dialog-service-opener').on('click',function() {
-    self._toggleServices();
-  });
+ReconDialog.prototype._nextDialog = function(column,selectedServiceRecordindex,serviceRecords,record) {
```

Comment:
Style suggestion: insert spaces after the `,` that separates arguments of a function.
```suggestion
ReconDialog.prototype._nextDialog = function(column, selectedServiceRecordindex, serviceRecords, record) {
```